### ISSUE TITLE
Prefer consecutive matches

### ISSUE THREAD

**Issue by xeruf** [jhawthorn/fzy#173]
![image](https://user-images.githubusercontent.com/13354331/170893534-ed4d5d71-981b-480e-bc92-9f99704d8cc7.png)

I would have expected "Walls" to match higher here, and especially "Walls Crumble" as it contains the search term twice, which apparently is not a case considered at all so far.

**Comment by jhawthorn** [comment]
Yes, I think this is working as intended. There's certainly some subjectivity in what algorithm and scores one prefers. fzy favours start-of-word matches (including acronym-style-matching) it sounds like you strictly want contiguous strings instead of fzy's more advanced scoring (I'm certain there are tools which do this)

> it contains the search term twice, which apparently is not a case considered at all so far.

I do not want a term occurring twice to be favoured, as that will unreasonably prioritize inputs with a repeating term - the user can always enter the same term twice, but have no way to say "I want to match 'alls' but only once"
